core-plugins/docs/FTP-batchsource.md

```diff
@@ -14,25 +14,20 @@ This source is used whenever you need to read from an FTP or SFTP server.
 
 Properties
 ----------
-**referenceName:** This will be used to uniquely identify this source for lineage, annotating metadata, etc.
+**Reference Name:** Name used to uniquely identify this sink for lineage, annotating metadata, etc.
 
-**path:** Path to file(s) to be read. The path uses filename expansion (globbing) to read files.
+**Path:** Path to file(s) to be read. The path uses filename expansion (globbing) to read files.
 Path is expected to be of the form prefix://username:password@hostname:port/path (Macro-enabled)
 
-**fileRegex:** Regex to filter out files in the path. It accepts regular expression which is applied to the complete
+**Regex Path Filter:** Regex to filter out files in the path. It accepts regular expression which is applied to the complete
 path and returns the list of files that match the specified pattern. (Macro-enabled)
 
-**fileSystemProperties:** A JSON string representing a map of properties
+**File System Properties:** A JSON string representing a map of properties
 needed for the distributed file system. (Macro-enabled)
 
-**inputFormatClassName:** Name of the input format class, which must be a subclass of FileInputFormat.
-Defaults to CombineTextInputFormat. (Macro-enabled)
-
-**ignoreNonExistingFolders:** Identify if path needs to be ignored or not, for case when directory or file does not
+**Ignore Non-Existing Folders:** Identify if path needs to be ignored or not, for case when directory or file does not
```

Comment:
I've been calling this 'Allow Empty Input' in the GCS, S3, and File sources so let's use that here too.